it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/radiobutton/basic/.content.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:deviceGroups="[mobile/groups/responsive]"
+        cq:lastModified="{Date}2022-09-13T16:10:13.429+05:30"
+        cq:lastModifiedBy="admin"
+        cq:template="/conf/core-components-it/settings/wcm/templates/af-blank-v2"
+        jcr:language="en"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Adaptive Form V2 (IT)"
+        sling:resourceType="forms-components-examples/components/page">
+        <guideContainer
+                fd:version="2.0"
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="forms-components-examples/components/form/container"
+                fieldType="form">
+            <radiobutton
+                    jcr:created="{Date}2022-09-09T16:03:48.562+05:30"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2022-09-12T18:33:12.993+05:30"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Radio Button"
+                    sling:resourceType="core/fd/components/form/radiobutton/v1/radiobutton"
+                    assistPriority="caption"
+                    enabled="{Boolean}true"
+                    enum="[0,1]"
+                    enumNames="[Item 1,Item 2]"
+                    fieldType="radio-group"
+                    mandatoryMessage="This is a required radiobutton"
+                    name="radiobutton"
+                    orientation="vertical"
+                    textIsRich="[true,true]"
```

Comment:
`textIsRich` is not captured from dialog, is it required?

Reply:
nope , but it is getting there by default, I can remove it

Reply:
we should remove this.

Reply:
removed